lecture_notes/10 refactoring/7-optimized.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const shift = (offset) => (point) => {
+  const shifted = {
+    x: (point.x += offset.x),
+    y: (point.y += offset.y),
```

Comment:
Why do you use `+=` operator here?

Reply:
I fixed everything you mentioned, can I please get some more points now?